src/lexer.coffee

```diff
@@ -136,6 +136,12 @@ exports.Lexer = class Lexer
     if id is 'default' and @seenExport and @tag() in ['EXPORT', 'AS']
       @token 'DEFAULT', id
       return id.length
+    if id is 'do' and regExSuper = /^(\s*super)(?!\(\))/.exec @chunk[3...]
+      @token 'SUPER', 'super'
+      @token 'CALL_START', '('      
+      @token 'CALL_END', ')'
+      [input, sup] = regExSuper
+      return sup.length + 3
```

Comment:
When would `sup.length` not be 5? And 3 is the length of `do` plus a space? What if someone types `do super` with two or more spaces between the words?